lib/fika/parser_helper.ex

```diff
@@ -31,10 +31,20 @@ defmodule Fika.ParserHelper do
     do_to_ast({[new_left | rest], line}, :exp_bin_op)
   end
 
+  def do_to_ast({[left, bin_op, right | rest], line}, :exp_bin_op)
+      when bin_op in ["||", "&&"] do
+    new_left = {:call, {String.to_atom(bin_op), line}, [left, right], :kernel}
+    do_to_ast({[new_left | rest], line}, :exp_bin_op)
+  end
```

Comment:
Looks like this is same as the one for +, -, *, / ? How about we add the new operators to the existing guard clause?